packages/dd-trace/src/util.js

```diff
@@ -20,8 +20,20 @@ function isError (value) {
   return false
 }
 
+function parseRules (rules) {
+  if (Array.isArray(rules)) return rules
```

Comment:
In what case can this be an array?

Reply:
Not really sure, I think I was thinking in the case someone set the variable in the code itself? I guess still not a case that needs considering, so this line isn't needed (as noted below by you, and I do agree, this whole method isn't really needed here anyways)